package.json

```diff
@@ -64,31 +64,30 @@
     "beachball-check": "beachball check"
   },
   "dependencies": {
-    "jose": "^5.2.3",
+    "jose": "^5.9.6",
     "node-fetch": "^2.7.0"
   },
   "devDependencies": {
-    "@faker-js/faker": "^8.4.1",
-    "@types/express": "^4.17.21",
-    "@types/jest": "^29.5.12",
-    "@types/node": "^20.11.30",
-    "@types/node-fetch": "^2.6.11",
+    "@faker-js/faker": "^9.2.0",
+    "@types/express": "^5.0.0",
+    "@types/jest": "^29.5.14",
+    "@types/node": "^22.9.1",
+    "@types/node-fetch": "^2.6.12",
     "@types/pem": "^1.14.4",
     "@types/supertest": "^6.0.2",
-    "@typescript-eslint/eslint-plugin": "^7.4.0",
-    "@typescript-eslint/parser": "^7.4.0",
+    "@typescript-eslint/eslint-plugin": "^8.15.0",
+    "@typescript-eslint/parser": "^8.15.0",
     "beachball": "^2.41.0",
     "dotenv": "^16.4.5",
-    "eslint": "^8.57.0",
+    "eslint": "^8.57.1",
     "eslint-config-google": "^0.14.0",
     "eslint-config-prettier": "^9.1.0",
-    "eslint-plugin-prettier": "^5.1.3",
-    "express": "^4.19.2",
+    "eslint-plugin-prettier": "^5.2.1",
+    "express": "^4.21.1",
     "jest": "^29.7.0",
-    "lint-staged": "^15.2.2",
```

Comment:
ah looks like i forgot to remove this in my CI update. can you also remove its [config file](https://github.com/passageidentity/passage-flex-node/blob/main/.lintstagedrc.json)?